finality-provider/service/fp_instance.go

```diff
@@ -32,15 +32,16 @@ type FinalityProviderInstance struct {
 	btcPk *bbntypes.BIP340PubKey
 
 	fpState      *fpState
-	pubRandState *pubRandState
+	pubRandState *PubRandState
```

Comment:
Similar to the other comment, do you think we should get rid of this `pubRandState` field from the `FinalityProviderInstance`, and instead only rely on `rndCommitter` for pub rand operations?